### DISCUSSION TITLE
[v6] tinymce showing an upgrade button?

### DISCUSSION THREAD

**Discussion by fronbow** [Laravel-Backpack/community-forum#528] [General]
Hey All

Just testing my existing code with v6 and noticed the tinymce control is showing an Upgrade button in its' panel
![tinymce-upgrade-button](https://github.com/Laravel-Backpack/community-forum/assets/1417531/e4eeb3ce-2076-4de9-b458-7f736c92105b)

I've just had a brief look on their site and noticed they're doing a subscription model but no obvious sign on how to remove it, so I thought I'd post a heads up here!

Cheers

**Comment by karandatwani92** [comment]
Thanks @fronbow for highlighting this. I have created an issue about this to inform the team. We will have the update on this soon here. https://github.com/Laravel-Backpack/PRO/issues/163
